qa/common/gen_qa_models.py

```diff
@@ -2486,8 +2489,7 @@ def create_fixed_models(
         import torch
         from torch import nn
     if FLAGS.openvino:
-        from openvino.inference_engine import IENetwork
-        import ngraph as ng
```

Comment:
The nGraph API is deprecated and will be removed in the 2024.0 release. See https://github.com/opencv/opencv/issues/24589